mapdata/WarcraftLegacies/AbilityData/Skin/1261449537.json

```diff
@@ -63,7 +63,7 @@
       "Level": 1,
       "Id": 828536161,
       "Type": 3,
-      "Value": "This unit is surrounded by Tentacles which attack nearby enemy units. Each Tentacle has the same attack damage as the hero."
```

Comment:
Oops... This was definitely something I had meant to program, my bad. Maybe one day I'll put it back in lol.